lib/diagnostic.rb

```diff
@@ -100,13 +103,13 @@
 # In a Ruby string, write how you would remove the last two elements from this
 # array
 
-Diagnostic.remove_from_array = 'your answer here'
+Diagnostic.remove_from_array = arr[0,2]
```

Comment:
This would grab the first two elements. What you'd want to use is `.pop`. So, `arr.pop 2` to `pop` off the last two elements